script.js

```diff
@@ -1,4 +1,246 @@
+var userLossCount = 0;
+var drawCount = 0;
+var recentWinner = "";
+
+// assigning each number to an object
+var randomItem = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomNumber = randomInteger + 1;
+  var randomItem = "";
+
+  console.log("random item:");
+  if (randomNumber == 1) {
+    randomItem = "scissors";
+  }
+  if (randomNumber == 2) {
+    randomItem = "paper";
+  }
+  if (randomNumber == 3) {
+    randomItem = "stone";
+  }
+  console.log(randomItem);
+  return randomItem;
+};
+
+// NORMAL mode
+var playNormalMode = function (userName, input) {
+  if ((currentGameMode = "scissor paper stone game")) {
+    myOutputValue = `Welcome, Player ${userName}! <br> You are now in NORMAL mode of scissor paper stone. <br> Please enter 'scissors', 'paper', or 'stone' to play the game.`;
+  }
+
+  var computerChoose = randomItem();
+
+  // assigning emojis to the icon
+  var inputEmoji;
+  if (input == "scissors") {
+    inputEmoji = "✌";
+  } else if (input == "paper") {
+    inputEmoji = "✋";
+  } else if (input == "stone") {
+    inputEmoji = "✊";
+  }
+  var compEmoji;
+  if (computerChoose == "scissors") {
+    compEmoji = "✌";
+  } else if (computerChoose == "paper") {
+    compEmoji = "✋";
+  } else if (computerChoose == "stone") {
+    compEmoji = "✊";
+  }
+
+  // normal game mode logic
+  // (In normal mode) if both parties choose the same object, it's a draw
+  if (input == computerChoose) {
+    userWinCount = userWinCount;
+    userLossCount = userLossCount;
+    drawCount = drawCount + 1;
+    playCount = playCount + 1;
+    myOutputValue = `It is a DRAW. <br> You choose: ${input} ${inputEmoji}. <br> The computer chooses: ${computerChoose} ${compEmoji}. <br> Player ${userName}, you have played ${playCount} times. <br> You have ${userWinCount} wins, ${userLossCount} loses, and ${drawCount} draws with the computer.`;
+  } // (In normal mode) user beats computer when: scissors beats paper, paper beats stone, and stone beats scissors.
+  else if (
+    (input == "scissors" && computerChoose == "paper") ||
+    (input == "paper" && computerChoose == "stone") ||
+    (input == "stone" && computerChoose == "scissors")
+  ) {
+    userWinCount = userWinCount + 1;
+    userLossCount = userLossCount;
+    drawCount = drawCount;
+    playCount = playCount + 1;
+    myOutputValue = `You WIN! <br> You choose: ${input} ${inputEmoji}. <br> The computer chooses: ${computerChoose} ${compEmoji}. <br> Player ${userName}, you have played ${playCount} times. <br> You have ${userWinCount} wins, ${userLossCount} loses, and ${drawCount} draws with the computer.`;
+  } // (In normal mode) computer beats user when: scissors loses to stone, paper loses to scissors, and stone loses to paper.
+  else if (
+    (input == "scissors" && computerChoose == "stone") ||
+    (input == "paper" && computerChoose == "scissors") ||
+    (input == "stone" && computerChoose == "paper")
+  ) {
```

Comment:
as win and draws are already covered, this can be placed under an else without listing out all the conditionals